spice/lib/reg.py

```diff
@@ -48,7 +48,7 @@
 
 
 class IVmAction(interface.Interface):
-    def __call__():
+    def __call__(self):
```

Comment:
Hi.
Sorry, this is Zope interface.
Interfaces are defined without "self".
So, it should be without self. Maybe add a note?)

Something like:
class IVmAction(interface.Interface):
   def __call__():   # Zope Interfaces are defined without 'self'

to avoid next further fix )?